api/napi/__test__/js_value_conversion.spec.ts

```diff
@@ -236,7 +236,7 @@ test('get/set image properties', async (t) => {
 
     instance!.invoke("image-width", [0]);
     instance!.invoke("image-height", [0]);
-    instance!.setProperty("alt-image", new ImageData(image.bitmap.data, 64));
+    instance!.setProperty("alt-image", { width: 64, height: 64, data: new Uint8ClampedArray(image.bitmap.data) });
```

Comment:
I wonder: What's the difference, testing wise, between this line and line 231? Maybe this entire block isn't needed?

Reply:
Yes you're right. It's the same I have removed the test.

Reply:
You have removed the line, but not the entire block. I think the test will fail in the CI. You can also remove the corresponding bits in the .slint code if they are not needed anymore.

Reply:
is removed